spec/services/govspeak_service_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+RSpec.describe GovspeakService do
+  describe "#to_html" do
+    it "converts the provided text to HTML" do
+      expect(GovspeakService.new("## Hullo").to_html)
+        .to match(%{<h2 id="hullo">Hullo</h2>})
+    end
+
+    context "when a contact that isn't known is provided in the govspeak" do
```

Comment:
This took me a couple of readings - something about the 'is provided' bit. Perhaps just 'when we reference an unknown contact'?

Reply:
Thanks - I like your suggestion